core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -17,15 +17,17 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
-import java.util.List
 import java.util.concurrent.CountDownLatch
+import java.util.{List => JList}
 
 import scala.collection.JavaConversions._
 
-import org.apache.mesos.Protos.{FrameworkInfo, Resource, Status}
-import org.apache.mesos.{MesosSchedulerDriver, Scheduler}
-import org.apache.spark.Logging
+import com.google.common.base.Splitter
+import org.apache.mesos.Protos._
+import org.apache.mesos.{MesosSchedulerDriver, Protos, Scheduler}
 import org.apache.spark.util.Utils
```

Comment:
nit: add empty line between `org.apache.mesos` and `org.apache.spark`.  btw I highly recommend using intellij's plugin to order imports, as described here: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports
be sure to use the plugin, not the built-in feature of Intellij